src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,86 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = (props) => {
+  const {
+    total,
+    perPage,
+    currentPage,
+    onPageChange,
+  } = props;
+  const amountOfTabs = Math.ceil(total / perPage);
+  const tabs = getNumbers(1, amountOfTabs);
+
+  const handlePageChange = (tab: number) => {
```

Comment:
onPageChange should be triggered only if the page was changed. Add a condition to check if the currentPage is different from the clicked tab.